jaxrs-spec/src/main/asciidoc/boilerplate.adoc

```diff
@@ -16,24 +16,20 @@
 
 == Boilerplate
 
-WARNING: This document contains just boilerplate information. It explicitly is *not* a complete specification. In particular, it misses all normative text. To learn about compliance with this specification, refer to "JAX-RS: Java™ API for RESTful Web Services" (Version 2.1, Final Release July 13, 2017).
+WARNING: This document contains just boilerplate information. It explicitly is *not* a complete specification.
+In particular, it misses all normative text.
 
 
 === Scope
 
-*Jakarta RESTful Web Services* provides a specification document, TCK and foundational API to develop web services following the Representational State Transfer (REST) architectural pattern.
-
-
-=== Acronyms
-
-Acronyms used in this document MAY refer to Oracle / JCP technology.
-
-The acronym JAX-RS is reused with permission of Oracle and either refers to "Jakarta RESTful Web Services" or "Java™ API for RESTful Web Services", dependending on the context.
+*Jakarta RESTful Web Services* provides a specification document, TCK and foundational API to develop web services
+following the Representational State Transfer (REST) architectural pattern.
 
 
 === TCK
 
-NOTE: At time of writing, there is no standalone TCK for *Jakarta RESTful Web Services*. To proof compliance with this specification, pass the Jakarta EE Platform TCK instead.
+The TCK for this release is available as a ZIP file from Eclipse at the following location:
+https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8/
```

Comment:
There's no need to add a reference to the TCK in this spec.
This link does *not* point to the EFTL-licensed TCK that is required
for compatibility claims so it doesn't provide much value here.